test/src/common/harnessStorage.c

```diff
@@ -293,15 +293,15 @@ hrnStoragePutLog(const Storage *storage, const char *file, const Buffer *buffer,
         strCatFmt(log, "cmp[%s]", strZ(compressTypeStr(param.compressType)));
 
     // Add encryption detail
-    if (param.cipherType != cipherTypeNone)
+    if (param.cipherType != 0 && param.cipherType != cipherTypeNone)
     {
         if (param.cipherPass == NULL)
             param.cipherPass = TEST_CIPHER_PASS;
 
         if (param.compressType != compressTypeNone)
             strCatZ(log, "/");
 
-        strCatFmt(log, "enc[%s,%s]", strZ(cipherTypeName(param.cipherType)), param.cipherPass);
+        strCatFmt(log, "enc[%s,%s]", strZ(strIdToStr(param.cipherType)), param.cipherPass);
     }
 
     // Add a space if compression/encryption defined
```

Comment:
On the line below this (line 308 - I'm unable to click on it) it has:
```
if (param.compressType != compressTypeNone || param.cipherType != cipherTypeNone)
```

Shouldn't it be checking param.cipherType !=0 as well? 
```
if (param.compressType != compressTypeNone || (param.cipherType != 0 && param.cipherType != cipherTypeNone)
```

Reply:
Yes, it should. Fixed in f3a74498cc0.